modules/core/src/main/java/org/apache/ignite/cache/affinity/fair/FairAffinityFunction.java

```diff
@@ -102,6 +102,9 @@
     /** Optional backup filter. First node is primary, second node is a node being tested. */
     private IgniteBiPredicate<ClusterNode, ClusterNode> backupFilter;
 
+    /** Optional backup filter. First node is primary, second node is a node being tested.*/
```

Comment:
The doc has to be changed.

"Optional affinity backups filter. The first node is a node being tested, the second is a list of nodes that are already assigned for a given partition (the first node in the list is primary)."